migrations/versions/dbb9670cd902_.py

```diff
@@ -0,0 +1,34 @@
+"""empty message
+
+Revision ID: dbb9670cd902
+Revises: f63b9b9b645b
+Create Date: 2017-07-31 02:20:34.268384
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'dbb9670cd902'
+down_revision = 'f63b9b9b645b'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('sponsors', 'level',
```

Comment:
Sorry, missed that :grimacing: , should I delete the sponsors with invalid level or should they be changed to NULL ?